agreements/activity-types.md

```diff
@@ -0,0 +1,19 @@
+# Activity Types Agreement
+
+We have assessed that there are four types of activities that members carry out within Root Systems:
+
+## External
+
+This is contracting work with the outside world - usually software development, but could include any other types of consulting work.
+
+## Internal Investment
+
+This is either internal work towards development of a Root Systems product, or personal development / upskilling of a member (i.e. conferences, training).
+
+## Internal Administration
+
+This is time spent towards the administration of the business - typically work done as part of either a [coordinator](../roles/coordinator) or [director](../roles/director) role.
```

Comment:
i haven't heard of "internal administration" as an activity type until this. is it possible to provide context for how this differs from "internal investment" (i think i get it intuitively but maybe not)? as in, is invoicing clients an example of this? and i thought the goal is to split out as many "fine-grained" / "micro-role" admin tasks away from coordinators and directors except for the "coarse-grained" ones (otherwise all the roles will inevitably centralize in the coordinators and directors beyond what is helpful), so i wonder how that fits in with per-day activity types.